### DISCUSSION TITLE
How do I define that a freight customer has two loading tracks?

### DISCUSSION THREAD

**Discussion by AlexEhn** [tellurianinteractive/Tellurian.Trains.ModulesRegistryApp#117] [Q&A] [answered]
The highlighted field is throwing an error if I enter "21 & 22" in the field. 
![image](https://user-images.githubusercontent.com/57874863/131734789-9a8a80d0-ca98-46cd-98cd-57cd6814c968.png)

**Comment by fjallemark** [comment]
How you name tracks or loading areas is up to you. If your customer have several tracks there are some possibilities:
- If loading and unloading takes place on two or more tracks, simply name it as a loading area and arriving trains can use any of the tracks in the loading area.
- if loading is on one track/area and unloading in another track/area, then it is two separate cargo flows. 

It is recommended that the colours you choose for tracks/areas also are recognized on your station layout documentation. This helps other users to orientate and use the correct tracks.

**Reply by AlexEhn** [reply]
But the description says "Track no." I interpret that as the track number, not the name of the track.

**Reply by AlexEhn** [reply]
And if the purpose of the field is for a name, "21 & 22" should be a perfectly fine name.

The field validations and description needs to be adjusted.

**Comment by fjallemark** [comment, marked as answer]
I have tested with your customer. **21 & 22** is a valid name, but there was a lot of spaces that made the name too long. This was also noticed in the error message.

Also, why have track 21 when all cargo flows are at track 22? You don't have to specify and of the **Other...**-fields for the cargo flow if it is the same as the values for the customer and selected cargo. Nor do you have to specify colour if it is the same colour for the customer.

**Reply by AlexEhn** [reply]
> You don't have to specify and of the **Other...**-fields for the cargo flow if it is the same as the values for the customer and selected cargo. Nor do you have to specify colour if it is the same colour for the customer.

Aha! Read the [Station Customer Cargo Flow](https://github.com/tellurianinteractive/Tellurian.Trains.ModulesRegistryApp/wiki/Stations,-Tracks-and-Cargo-Customers#station-customer-cargo-flow) in the Wiki but it was not clear despite that since they are not referenced as "other... " in the wiki.

**Reply by fjallemark** [reply]
Better to read the help text in the Module Registry. Klick on the ? up in the right corner.

**Reply by AlexEhn** [reply]
Oh, did not understand the purpose of that icon. 

Perhaps can be highlighted with at text "field explanations in help button at upper right"

**Comment by AlexEhn** [comment]
Confirmed that "21 & 22" can be entered if extra blank spaces are removed.

**Comment by AlexEhn** [comment]
The freight customer has two tracks, 21 & 22, that holds two waggons each. It is up to the driver which waggon to put on track 21 or 22.  

Four waggons should be picked up and replaced by four new waggons by the morning switcher. In the afternoon the now empty waggons are replaced by four new waggons.
